test/integration/api/users.js

```diff
@@ -300,6 +300,21 @@ describe('api: /users', () => {
         .expect(401)));
     }));
 
+    it('should log action in audit log if password is invalidated', testService(async (service) => {
+      const asAlice = await service.login('alice');
```

Comment:
This is taking advantage of the new usage of `service.login()` that doesn't require a callback. Since I'm passing a single string to `service.login()`, not an array, I wanted `service.login()` to return a single authed service, not an array of a single authed service. I made a small change to test/integration/setup.js to facilitate that.